exercises/bob/bob.py

```diff
@@ -1,2 +1,2 @@
-def hey(string):
+def hey(incoming_words):
```

Comment:
Maybe just `words` or even `phrase` is a better choice?

Reply:
You're right, I agree with phrase, will recommit with that